src/main/java/org/terasology/fluid/event/BeforeFluidPutInInventory.java

```diff
@@ -18,31 +18,67 @@
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.AbstractConsumableEvent;
 
+/**
+ * This event indicates that an entity is putting a fluid in an inventory slot, and contains attributes indicating the
+ * state of the fluid before it was put in the inventory.
+ */
 public class BeforeFluidPutInInventory extends AbstractConsumableEvent {
+    /** The entity who is putting the fluid in the inventory */
     private EntityRef instigator;
+    /** The type of the fluid being put in the inventory */
     private String fluidType;
+    /** The volume of the fluid being put in the inventory */
     private float volume;
+    /** The slot number of the inventory slot in which the fluid is being put */
     private int slot;
 
+    /**
+     * Parametrized constructor.
+     *
+     * @param instigator the entity who is putting the fluid in the inventory
+     * @param fluidType  the type of the fluid being put in the inventory
+     * @param volume     the volume of the fluid being put in the inventory
+     * @param slot       the slot number of the inventory slot in which the fluid is being put
+     */
     public BeforeFluidPutInInventory(EntityRef instigator, String fluidType, float volume, int slot) {
         this.instigator = instigator;
         this.fluidType = fluidType;
         this.volume = volume;
         this.slot = slot;
     }
 
+    /**
+     * Accessor function that returns the fluid type.
+     *
+     * @return the fluid type of the fluid being put in the inventory
+     */
     public String getFluidType() {
         return fluidType;
     }
 
+    /**
+     * Accessor function that returns the instigator of the action.
+     *
+     * @return an EntityRef to the entity who is putting the fluid in the inventory
```

Comment:
who → which, putting → placing. There are a few more of these so I won't list them all - do a ctrl+f for "who" and "put", replace where necessary